core/src/main/java/org/apache/carbondata/core/metadata/blocklet/index/BlockletIndex.java

```diff
@@ -75,4 +80,12 @@ public void setMinMaxIndex(BlockletMinMaxIndex minMaxIndex) {
     this.minMaxIndex = minMaxIndex;
   }
 
+  @Override public void write(DataOutput out) throws IOException {
```

Comment:
Please don't serialize as it is not required to pass to executor

Reply:
ok